Update Unbound to v1.13.2

mxmartins opened the pull request.

MatthewVance commented:
I appreciate this. I'm going to close instead of merge because I released some additional updates alongside 1.13.2. I updated to Debian Bullseye, made use of the new with-deprecate-rsa-1024 build option, and also set the new hide-http-user-agent and http-user-agent configuration options. I changed how the Labels are set as well.

mxmartins commented:
Even better, thank you!

 

Another user out there reached out to me and was looking to “changed OpenSSL and Unbound compilation to allow the usage of all processors on build, leading to a better build timing”….  I told them to submit the request to you as opposed to try to merge it to my PR…  Not sure if they ever reached out to you.

 

Thank for keeping this updated….

 

From: Matthew Vance ***@***.***> 
Sent: Saturday, August 21, 2021 1:53 PM
To: MatthewVance/unbound-docker ***@***.***>
Cc: mxmartins ***@***.***>; Author ***@***.***>
Subject: Re: [MatthewVance/unbound-docker] Update Unbound to v1.13.2 (#72)

 

I appreciate this. I'm going to close instead of merge because I released some additional updates alongside 1.13.2. I updated to Debian Bullseye, made use of the new with-deprecate-rsa-1024 build option, and also set the new hide-http-user-agent and http-user-agent configuration options. I changed how the Labels are set as well.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub <https://github.com/MatthewVance/unbound-docker/pull/72#issuecomment-903168551> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/AL7ZG7524EZW67DB3WI6GA3T5773LANCNFSM5CBQQO2A> .
Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>  or Android <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email> .  <https://github.com/notifications/beacon/AL7ZG742MQ6JO2LRG4RYJY3T5773LA5CNFSM5CBQQO2KYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOGXKUEJY.gif>
